Doc/library/urllib.request.rst

```diff
@@ -1323,7 +1329,10 @@ Use the *headers* argument to the :class:`Request` constructor, or::
    req.add_header('Referer', 'http://www.python.org/')
    # Customize the default User-Agent header value:
    req.add_header('User-Agent', 'urllib-example/0.1 (Contact: . . .)')
-   r = urllib.request.urlopen(req)
+   try:
```

Comment:
Sorry, I mean all three. One is the example of using this without `with`, which is fine. But what about these two?

It does not look like they are also examples without context managers.